torch/_inductor/fx_passes/post_grad.py

```diff
@@ -1202,6 +1202,14 @@ def view_to_reshape(gm):
     """
     Replace view ops in the GraphModule to reshape ops.
     """
+    subgraph_names: OrderedSet[str] = OrderedSet(
+        x.target for x in gm.graph.find_nodes(op="get_attr")
+    )
+
+    for child_name, child_mod in gm.named_children():
+        if child_name in subgraph_names and isinstance(child_mod, torch.fx.GraphModule):
+            view_to_reshape(child_mod)
```

Comment:
I would prefer some automated way of doing this, like GraphTransformObserver accepts `view_to_reshape` and then automatically applies the pass to subgraphs.

How many more times do we need to do this manually?